types/setting.go

```diff
@@ -195,6 +199,28 @@ var (
 		SettingNameBackingImageCleanupWaitInterval:              SettingDefinitionBackingImageCleanupWaitInterval,
 		SettingNameGuaranteedEngineManagerCPU:                   SettingDefinitionGuaranteedEngineManagerCPU,
 		SettingNameGuaranteedReplicaManagerCPU:                  SettingDefinitionGuaranteedReplicaManagerCPU,
+		SettingNameSupportBundleImage:                           SettingDefinitionBundleImage,
+		SettingNameSupportBundleImagePullPolicy:                 SettingDefinitionSupportBundleImagePullPolicy,
+	}
+
+	SettingDefinitionBundleImage = SettingDefinition{
+		DisplayName: "Support Bundle Image Name",
+		Description: "The image name of the support bundle",
+		Category:    SettingCategoryGeneral,
+		Type:        SettingTypeString,
+		Required:    false,
+		ReadOnly:    false,
+		Default:     "rancher/harvester-support-bundle-utils:master-head",
```

Comment:
I see why the image pull policy is an option now. But don't use `master-head`. We need to release (and it cannot be called `harvester-support-bundle-utils`) and pin down a released version, otherwise, API might be imcompatible.

Reply:
Updated to use the existing image pull policy